frontend/src/config/constants.js

```diff
@@ -24,5 +24,6 @@ export const lightGrey = "#565652";
 export const darkGrey = "#323739";
 export const yellow = "#FAAD14";
 export const purple = "#9254de";
+export const lightPurple = "#efdbff";
```

Comment:
is light purple used anywhere LOL. If not remove it?

Reply:
it should be 😡😡😡  (term planner 👀)